config/services.yaml

```
@@ -36,6 +37,10 @@ services:
     # add more service definitions when explicit configuration is needed
     # please note that last definitions always *replace* previous ones
 
+    App\Service\ResourceService:
+        arguments:
+            $excludesResources: '%excluded_resources%'
```

Typo if you wish to fix.
```suggestion
            $excludedResources: '%excluded_resources%'
```